<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>51</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actRecInsert"/>
   <addaction name="separator"/>
   <addaction name="actRecEdit"/>
   <addaction name="separator"/>
   <addaction name="actRecDelete"/>
   <addaction name="separator"/>
   <addaction name="actScan"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
   <addaction name="separator"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actRecInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/insert.png</normaloff>:/images/insert.png</iconset>
   </property>
   <property name="text">
    <string>插入记录</string>
   </property>
  </action>
  <action name="actRecDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actRecEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑记录</string>
   </property>
  </action>
  <action name="actScan">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/upSalary.png</normaloff>:/images/upSalary.png</iconset>
   </property>
   <property name="text">
    <string>涨工资</string>
   </property>
  </action>
  <action name="actSQL">
   <property name="text">
    <string>SQL测试</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
